### DISCUSSION TITLE
Cannot create path aliases

### DISCUSSION THREAD

**Discussion by MaksimPrudnikau** [remix-run/remix#8085] [Q&A]
Hello! I tried to add some path aliases, but they could not be found. Here is my tsconfig.json
```ison
"paths": {
      "~/*": ["./app/*"],
      "~/lib": ["./lib/*"]
 },
```

This code didn't not work `TS2307: Cannot find module  ~/lib/types/Pages  or its corresponding type declarations`
```
import { Pages } from "~/lib/types/Pages";
```

here is my project structure
```
app/
build/
lib/
  types/
     Pages.ts
     db.ts
  hooks/
   ...
public
```

**Comment by kiliman** [comment]
I think it needs to be
```json
"paths": {
      "~/*": ["./app/*"],
      "~/lib/*": ["./lib/*"]
 },
```
